docker/suricata/dist/suricata.yaml

```diff
@@ -399,7 +403,7 @@ outputs:
 
   # Stats.log contains data from various counters of the Suricata engine.
   - stats:
-      enabled: no
+      enabled: yes
```

Comment:
I enabled this because of the error below in my logs, but we can disable it altogether if not interested:
```
<Error> - [ERRCODE: SC_ERR_STATS_LOG_GENERIC(278)] - eve.stats: stats are disabled globally: set stats.enabled to true.